gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/platform/LicenseResource.java

```diff
@@ -38,21 +39,22 @@ public class LicenseResource extends AbstractResource {
     @Autowired
     private LicenseManager licenseManager;
 
+    @Value("${notifiers.licenseExpiration.enabled:true}")
+    private boolean isLicenseExpirationNotifierEnabled = true;
+
     @GET
     @Operation(summary = "Get current node License")
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
     public void get(@Suspended final AsyncResponse response) {
         final License platformLicense = licenseManager.getPlatformLicense();
-        GraviteeLicense license;
-        license = GraviteeLicense.builder()
+        GraviteeLicense license = GraviteeLicense.builder()
                 .tier(platformLicense.getTier())
                 .packs(platformLicense.getPacks())
                 .features(platformLicense.getFeatures())
+                .expirationDate(isLicenseExpirationNotifierEnabled ? platformLicense.getExpirationDate() : null)
```

Comment:
**suggestion**: in order to manage the notification based on the role, maybe we should provide a new method into the "permissionService" to check if a user has at least a role in a given list of role. If the user has this role, we should set the expirationDate attribute.

Reply:
notification will be shown based a new permission.